instrumentation/README.md

```diff
@@ -1,181 +1,247 @@
+- RPM with `dnf`:
+    ```shell
+    cat <<EOH > /etc/yum.repos.d/splunk-otel-collector.repo && dnf install -y splunk-otel-systemd-auto-instrumentation
+    [splunk-otel-collector]
+    name=Splunk OpenTelemetry Collector Repository
+    baseurl=https://splunk.jfrog.io/splunk/otel-collector-rpm/release/\$basearch
+    gpgcheck=1
+    gpgkey=https://splunk.jfrog.io/splunk/otel-collector-rpm/splunk-B3CD4420.pub
+    enabled=1
+    EOH
+    ```
+- RPM with `zypper`:
+    ```shell
+    cat <<EOH > /etc/zypp/repos.d/splunk-otel-collector.repo && zypper install -y splunk-otel-systemd-auto-instrumentation
+    [splunk-otel-collector]
+    name=Splunk OpenTelemetry Collector Repository
+    baseurl=https://splunk.jfrog.io/splunk/otel-collector-rpm/release/\$basearch
+    gpgcheck=1
+    gpgkey=https://splunk.jfrog.io/splunk/otel-collector-rpm/splunk-B3CD4420.pub
+    enabled=1
+    EOH
+    ```
+
+### Debian and RPM Packages
+
+Download and install the `splunk-otel-systemd-auto-instrumentation` package ***without*** setting up a
+[package repository](#debian-and-rpm-repositories) (requires `root` privileges):
+1. Download the appropriate `splunk-otel-systemd-auto-instrumentation` deb/rpm package for the target system
+   (amd64/x86_64 or arm64/aarch64) from [GitHub Releases](https://github.com/signalfx/splunk-otel-collector/releases).
+2. Download and install the public key for package signature verification:
+   - Debian:
+       ```shell
+       curl -sSL https://splunk.jfrog.io/splunk/otel-collector-deb/splunk-B3CD4420.gpg > /etc/apt/trusted.gpg.d/splunk.gpg
+       ```
+   - RPM:
+       ```shell
+       rpm --import https://splunk.jfrog.io/splunk/otel-collector-rpm/splunk-B3CD4420.pub
+       ```
+3. Install the package with the following command (replace `<path>` with the local path to the
+   downloaded package):
+   - Debian:
+       ```shell
+       $ dpkg -i <path>
+       ```
+   - RPM:
+       ```shell
+       $ rpm -ivh <path>
+       ```
+
+## Configuration
+
+See the [Advanced Configuration Guide](
+https://docs.splunk.com/Observability/gdi/get-data-in/application/java/configuration/advanced-java-otel-configuration.html)
+for details about supported options and defaults for the Java agent.  These options can be configured via
+[environment variables](#systemd-environment-variables) or their corresponding [system properties](#configuration-file)
+after installation.
+
+> ### Configuration Priority
+> The Java agent can consume configuration from one or more of the following sources
+> (ordered from highest to lowest priority):
+> 1. Java system properties (`-D` flags) passed directly to the agent.  For example,
+>      ```shell
+>      JAVA_TOOL_OPTIONS="-javaagent:/usr/lib/splunk-instrumentation/splunk-otel-javaagent.jar -Dotel.service.name=my-service"
+>      ```
+> 2. Environment variables
+> 3. Configuration files
+> 
+> It may be necessary to check the configuration of the system or individual services for potential conflicts before
```

Comment:
I'd be more direct: "Before making any changes, check the configuration... for potential conflicts.